integration/cli-tests/src/test/java/EnvSetup.sh

```diff
@@ -26,56 +26,23 @@ BASEDIR=$(dirname "$0")
 echo "Platform : $platform"
 echo "BitType : $bitType"
 
-getPomVersion(){
-    VERSION=$(cat pom.xml | grep "^    <version>.*</version>$" | awk -F'[><]' '{print $3}');
-    echo "Version : $VERSION"
-}
-
-buildCli(){
-    go mod vendor
-    sleep 10
-
-    #build Micro Integrator CLI
-    echo "Build Micro Integrator CLI"
-    cd ../
-
-    #get the version from the pom
-    getPomVersion
-    cd cmd
-    ./build.sh -t mi.go -v ${VERSION} -f
-}
-
 #Setting up the CLI environment
 #Check if the cli build is available in the location
-setup(){
-
 cd $BASEDIR
 
-DIR="../../../../../cmd/build"
+#Get the CLI distribution from the latest jenkins build
+DISTRIBUTION_NAME=$(eval "curl -s https://wso2.org/jenkins/job/products/job/product-mi-tooling/lastSuccessfulBuild/api/json | python3 -c \"import sys, json; print(json.load(sys.stdin)['artifacts'][1]['fileName'])\"")
```

Comment:
This is going to run this only in linux,   If we can consider platform and bit type while downloading then we can run in others as well

Reply:
I did not want to do it since the purpose of this change is different, let's consider it as a next step. WDYT?

Reply:
+1